src/types.mligo

```diff
@@ -151,10 +151,15 @@ type last_period_change =
 
 // -- Storage -- //
 
+type governance_token =
+  { address : address
+  ; token_id : nat
```

Comment:
LGTM, but we also have the `token_id` type, which is just a type synonym for `nat` (and will result in the same  code, but since we have it let's use that.

Reply:
Have changed.